boss-ws/src/main/java/org/genomebridge/boss/http/models/ResolveRequest.java

```diff
@@ -0,0 +1,19 @@
+package org.genomebridge.boss.http.models;
+
+
+import com.wordnik.swagger.annotations.ApiModel;
+import com.wordnik.swagger.annotations.ApiModelProperty;
+
+@ApiModel("Resolve Request")
+public class ResolveRequest {
+
+    @ApiModelProperty(value = "value must be strictly positive",dataType = "Integer")
+    public Integer validityPeriodSeconds;
+    @ApiModelProperty(value="'httpMethod' are one of { GET, PUT, HEAD }",dataType = "String")
```

Comment:
Language: "The 'http' method to use for accessing this object. Must be one of the following: GET, PUT, HEAD."